### PR TITLE
756 workflow template adjustments

### PR DESCRIPTION
**Pull request by pchelle**

### CODE REVIEW

**Comment by Yuri05** [comment]
All Linux  builds fail because ospsuite.utils and patchwork are not installed

File: R/utilities-writing-report.R [hunk 1]
```diff
@@ -445,3 +447,23 @@ getSimulationDescriptor <- function(workflow) {
...
+
+#' @title adjustTitlePage
+#' @description Adust Qualification Version Information to be displayed on title page 
+#' @param fileName name of .md file to update
+#' @param qualificationVersionInfo A `QualificationVersionInfo`object defining Qualification Version Information to be displayed on title page
+#' @export
+adjustTitlePage <- function(fileName, qualificationVersionInfo = NULL) {
+  validateIsOfType(qualificationVersionInfo, "QualificationVersionInfo", nullAllowed = TRUE)
+  # Does not adust title page if no QualificationVersionInfo
+  if(isEmpty(qualificationVersionInfo)){
```
Comment (Yuri05) [inline R/utilities-writing-report.R:459]: isEmpty or isNull?
Or is it the same here?

Comment (pchelle) [reply R/utilities-writing-report.R:459]: They will work the same way here

File: inst/extdata/qualification-workflow-template.R [hunk 2]
```diff
@@ -61,6 +77,10 @@ createQualificationReport <- function(qualificationRunnerFolder,
   #' If not set, report created will be named `report.md` and located in  `reOutputFolder`
   reportName <- file.path(reOutputFolder, "report.md")
   
+  #' Provide a template docx document passed to Pandoc for the conversion of the md report into docx
+  #' Default template is available in `system.file("extdata", "reference.docx", package = "ospsuite.reportingengine")`
+  wordConversionTemplate <- NULL
```
Comment (Yuri05) [inline inst/extdata/qualification-workflow-template.R:82]: Let's provide wordConversionTemplate as an argument of createQualificationReport

File: inst/extdata/qualification-workflow-template.R [hunk 3]
```diff
@@ -94,6 +114,12 @@ createQualificationReport <- function(qualificationRunnerFolder,
   }
 
   #-------- STEP 3: Run Qualification Workflow  --------#
+  # If version info is provided update title page
+  if(!is.null(versionInfo)){
+    titlePageFile <- file.path(reInputFolder, "Content/titlepage.md") 
+    adjustTitlePage(titlePageFile, qualificationVersionInfo = versionInfo)
```
Comment (Yuri05) [inline inst/extdata/qualification-workflow-template.R:119]: Let's check if the title page file exists, otherwise adjustTitlePage would crash

**Pull request merged**